test/spec/estimatedScore/estimatedScoreEventsHandlerSrv.spec.js

```diff
@@ -0,0 +1,290 @@
+describe('testing service "EstimatedScoreEventsHandlerSrv":', function () {
+    'use strict';
+
+    beforeEach(module('znk.infra.estimatedScore', 'htmlTemplates', 'testUtility', 'storage.mock'));
+
+    var rawPointsForExerciseTypeMap = {
+        4:{
```

Comment:
For the test is enough to no check all the exercises since the behaviour is the same.